test/crypto.js

```diff
@@ -2,215 +2,233 @@
 /* global MasqCommon */
 /* global chai */
 
+const { ERRORS } = MasqCommon.errors
+const { crypto } = MasqCommon
```

Comment:
+1

Reply:
if you have other problems with globals, you should be able to declare these var locally inside the describe section